device/arista/x86_64-arista_7060x6_64pe/Arista-7060X6-64PE-256x200G/BALANCED/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,36 @@
+{%- set default_cable = '5m' %}
+
+{%- include 'buffer_ports.j2' %}
+
+{%- macro generate_buffer_pool_and_profiles() %}
+    "BUFFER_POOL": {
+        "ingress_lossless_pool": {
+            "size": "121964196",
+            "type": "ingress",
+            "mode": "dynamic",
+            "xoff": "29520896"
+        },
+        "egress_lossless_pool": {
```

Comment:
should we just say this is egress_pool?

Reply:
i do not know if other repo (swss/sonic-utilitie) make any assumption on the name, we need to check that.

Reply:
yes, things like qos tests are currently assuming the name while grabing the configuration, changing the name will cause these tests to break at this moment...